travel-planner/travel-planner-ui/src/Components/Loading/Loading.jsx

```diff
@@ -0,0 +1,19 @@
+import "./Loading.css";
+
+export default function Loading() {
+	return (
+		<div className="loading-div">
+			<h1 className="loading">Loading...</h1>
+			<div className="lds-roller">
+				<div></div>
+				<div></div>
+				<div></div>
+				<div></div>
+				<div></div>
+				<div></div>
+				<div></div>
+				<div></div>
```

Comment:
What's that? Why we need a list of empty divs?

Reply:
This is creating the loading circles on the loading screen.

Reply:
Whoa, that's hacky. Probably we can think about refactor it after finish building the main features. Like https://contactmentor.com/how-to-add-loading-spinner-react-js/